app/views/ui-components/v1/cards/_eligibility_check.html.slim

```diff
@@ -0,0 +1,113 @@
+.panel.panel-default#employee-enrollments
+  .panel-body
+    .run-eligibility-check-btn-container        
+      = button_to l10n("employers.plan_years.eligibility_button_text"), '/', class: "btn btn-default", id: "eligibilityCheckButton", data: { url: profiles_employers_employer_profile_run_eligibility_check_path(@employer_profile) }
+    .col-xs-12.loading.run-eligibility-processing.hide
+      i.fa.fa-spinner.fa-spin.fa-2x
+    .run-eligibility-check-response-container(style="display: none;")
+      span.eligibility-response-close-icon
+        | X
+      .run-eligibility-check-response
+        p.eligibility-status-text.minimum-participation
+          => l10n("employers.plan_years.minimum_participation_text")
+          | &nbsp;&nbsp;
+          span.not-eligible
+            | &#10060;
+          span.is-eligible-checkmark style="display:none;"
+            | &#10004;
+        p.eligibility-status-text.non-business-owner-eligibility-count
+          => l10n("employers.plan_years.non-business_owner_eligibility_count_text")
+          | &nbsp;&nbsp;
+          span.not-eligible
+            | &#10060;
+          span.is-eligible-checkmark style="display:none;"
+            | &#10004;
+        p.eligibility-status-text.minimum-eligible-member-count
+          => l10n("employers.plan_years.minimum_eligible_member_count_text")
+          | &nbsp;&nbsp;
+          span.not-eligible
+            | &#10060;
+          span.is-eligible-checkmark style="display:none;"
+            | &#10004;
+
+
+javascript:
+  document.getElementById('eligibilityCheckButton').addEventListener('click', function() {
+      this.disabled = true;
+      
+      // Show Spinner
+      $('#eligibilityCheckButton').hide();
+      $('.run-eligibility-processing').removeClass("hide");
```

Comment:
I think this can be replaced with .show() for readability, but I know it will work anyway